Source/WebCore/PAL/pal/PALSwift.h

```diff
@@ -28,8 +28,39 @@
 #include <cstdint>
 #include <wtf/Vector.h>
 
+namespace Cpp {
+
 using VectorUInt8 = WTF::Vector<uint8_t>;
 using SpanConstUInt8 = std::span<const uint8_t>;
+using OptionalVectorUInt8 = std::optional<WTF::Vector<uint8_t>>;
+
+
+// FIXME: remove when swift support is available rdar://118026392
+inline OptionalVectorUInt8 toOptionalVectorUInt8(VectorUInt8 val)
+{
+    return val;
+}
+
+inline VectorUInt8 createVectorUInt8FromSpan(SpanConstUInt8 span)
```

Comment:
makeVectorUInt8.

Does the Vector(std::span<const T>) constructor not work in Swift? If so, that probably deserves a Radar and a comment linking to the Radar.